corehq/apps/userreports/reports/builder/columns.py

```diff
@@ -311,9 +311,14 @@ def _get_aggregation_config(self, agg):
 
     def get_indicator(self, aggregation, is_multiselect_chart_report=False):
         return {
+            "column_id": "count",
             "display_name": "Count",
-            "type": "count",
-            "column_id": "count"
+            "type": "expression",
+            "datatype": "integer",
```

Comment:
Currently counts look to be based on booleans https://github.com/dimagi/commcare-hq/blob/a01184c805f1d67be19f9780ba9036eff346c6e2/corehq/apps/userreports/indicators/factory.py#L27

Any reason you changed it to an integer? (I think that's probably fine since it's meant as a sum, but I'm not sure how much test coverage we have around this, so would default to keeping it the same)

Reply:
That makes sense.  Is there a quick way to create a boolean indicator that always returns true?

A hacky way would do something like:
```
"filter": {
   "type": "boolean_expression",
   "operator": "eq",
   "expression": {
      "type": "constant",
      "constant": 1
   },
   "property_value": 1
}
```